### PR TITLE
Streaming scss changes with browsersync

### PR DESCRIPTION
**Pull request by jonathanjfshaw**
A new guide for streaming scss changes with browsersync.

### CODE REVIEW

File: docs/guides/streaming-scss-changes-with-browsersync-gulp.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+[Browsersync] Proxying: http://appserver_nginx
+[Browsersync] Access URLs:
+  Local: http://localhost:3000
+  External: http://172.20.0.6:3000
+```
+* Visit your browsersync url https://bs.myproject.lndo.site in your browser. It should load normally.
+* In the source of this page search for "browser-sync". You should see the browser-sync-client script has been injected into the page.
+* Run `lando logs -s node` again. You should see something like `[Browsersync] Browser Connected: Chrome, version: 80.0.3987.149`.
+* In your IDE edit a style rule in an scss file, save if necessary, and wait a few seconds.
+* Run `lando logs -s node` again. You should see something like:
```
Comment (serundeputy) [inline docs/guides/streaming-scss-changes-with-browsersync-gulp.md:156]: similar here `observe the logs`

File: docs/guides/streaming-scss-changes-with-browsersync-gulp.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+      email: alliance@lando.dev
+      link: https://github.com/jonathanjfshaw
+---
+
+# Streaming scss changes with Browsersync & Gulp
+
+<GuideHeader test="" name="Jonathan Shaw" pic="https://www.gravatar.com/avatar/ba46283b2033d3a0201f2042746f81e7" link="https://github.com/jonathanjfshaw" />
+<YouTube url="" />
+
+Browsersync (https://www.browsersync.io/) is a nodejs package that synchronizes website changes to the browser.
```
Comment (serundeputy) [inline docs/guides/streaming-scss-changes-with-browsersync-gulp.md:32]: Make this a link: `[Browsersync](https://www.browsersync.io)`

File: docs/guides/how-to-wire-up-browsersync-and-gulp-in-my-lando-app.md [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+
+### Stream changes whenever Lando is running
+It's most convenient if Browsersync is running in the background whenever your Lando site is up. 
+One simple way to achieve this is to fire off a gulp task whenever Lando starts up. 
+If you had a gulp task called 'watch' then you'd put this in .lando.yml:
+```yaml
+services:
+  node:
+    type: node
+    command: cd /app/path/to/gulp/package/gulp watch
```
Comment (serundeputy) [inline docs/guides/how-to-wire-up-browsersync-and-gulp-in-my-lando-app.md:58]: for this i needed:

```yaml
commmand: cd /app/path/to/gupfile && node_modues/.bin/gulp watch
```

File: docs/guides/how-to-wire-up-browsersync-and-gulp-in-my-lando-app.md [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+## Configuring Lando for Browsersync
+
+### Create the proxy
+
+First you must create a proxy domain for browsersync to use. Changes you make to your scss files will only show up when
+you are looking at this https://bs.myproject.lndo.site, not at the usual domain https://myproject.lndo.site. 
+Add your equivalent of this to your .lando.yml file:
+```yaml
+proxy:
+    - bs.myproject.lndo.site:3000
```
[the remark this answers is not in the document]
Comment (jonathanjfshaw) [reply docs/guides/how-to-wire-up-browsersync-and-gulp-in-my-lando-app.md]: Whoops!

File: docs/guides/how-to-wire-up-browsersync-and-gulp-in-my-lando-app.md (hunk 3 above)
Comment (jonathanjfshaw) [reply docs/guides/how-to-wire-up-browsersync-and-gulp-in-my-lando-app.md]: My bad, that is what I'm actually using. I just (wrongly) thought the other would be OK and forgot to test it.

**Comment by serundeputy** [comment]
@jonathanjfshaw re-rolling over here https://github.com/lando/lando/pull/2145
w/ latest master
framing as question to be consistent with other guides.

**Pull request closed without merging**